pydatarecognition/cif_io.py

```diff
@@ -355,7 +355,5 @@ def print_story(user_input, args, ciffiles, skipped_cifs):
     print(f'Done working with cifs.\n{frame_dashchars}\nGetting references...')
 
 if __name__=="__main__":
-    import pathlib
-    toubling_path = pathlib.Path(os.path.join(os.pardir, 'docs\\examples\\cifs\\kd5015Mg3OH5Cl-4H20sup2.rtv.combined.cif'))
-    po = cif_read(toubling_path)
-    po.q
+    from pathlib import Path
+    json_dump(powdercif_to_json(cif_read(Path('../docs/examples/cifs/calculated/bs0018IIIsup4.rtv.simulated.cif'))), Path('../test.json'))
```

Comment:
This part was originally used to generate a test cif file in json format so that I can use it for testing but I forgot to delete it. I think the cif_to_json file now returns cif file in a doc form (json form).!